### ISSUE TITLE
fine tune

### ISSUE THREAD

**Issue by alicera** [ultralytics/yolov5#1850]
How to fine tune the epoch 300 model

**Comment by glenn-jocher** [comment]
@alicera you simply start training from your trained model:

`python train.py --weights your_model.pt ...`

**Comment by alicera** [comment]
python -m torch.distributed.launch --nproc_per_node 4 train.py --batch-size 128 --data ./data/coco.yaml --cfg ./models/yolov5s.yaml --weights your_model.pt

Can it finetune?

**Comment by alicera** [comment]
For example,
I command this and it will generate the 14.1MB .pt 
python -m torch.distributed.launch --nproc_per_node 4 train.py --batch-size 128 --data ./data/coco.yaml --cfg ./models/yolov5s.yaml --weights ''

Then, I command this with the above model  and it will generate the 56.8MB .pt
 python -m torch.distributed.launch --nproc_per_node 4 train.py --batch-size 128 --data ./data/coco.yaml --cfg ./models/yolov5s.yaml --weights last.pt

It should be the 14.1MB for the second commad, right?

**Comment by glenn-jocher** [comment]
> python -m torch.distributed.launch --nproc_per_node 4 train.py --batch-size 128 --data ./data/coco.yaml --cfg ./models/yolov5s.yaml --weights your_model.pt
> 
> Can it finetune?

Sure.

**Comment by alicera** [comment]
For the finetune , how to design the learning rate

**Comment by alicera** [comment]
Hi, glenn-jocher 
How to fine tune with last conv and fix the previous conv ?

**Comment by glenn-jocher** [comment]
I don't know what you're asking.

**Comment by alicera** [comment]
Like this
https://www.pyimagesearch.com/2019/06/03/fine-tuning-with-keras-and-deep-learning/
![image](https://user-images.githubusercontent.com/59388989/106413111-3dc95f00-6484-11eb-978a-062e64e3def5.png)

Does yolov5 have the method to fine-tune?

**Comment by glenn-jocher** [comment]
@alicera steps 1 and 2 above are handled automatically during YOLOv5 training, no action is required on your part. If you'd like to freeze layers when training see the Transfer Learning with Frozen Layers  ⭐ NEW tutorial:
https://docs.ultralytics.com/yolov5

**Comment by alicera** [comment]
1.
If I alredy train a model "last.pt" and its epoch is 300. I use below command to train other new data.
Is the process resume network?  The final last.pt will focus the new data,right. 

$ python -m torch.distributed.launch --nproc_per_node 4 train.py --batch-size 128 --data ./data/coco.yaml --cfg ./models/yolov5s.yaml --weights last.pt 

2. How about fine-tune dataset number ?
     It will be good.

**Comment by glenn-jocher** [comment]
@alicera omit --cfg, it's not needed.
